pkg/apis/core/v1/defaults.go

```diff
@@ -363,8 +363,8 @@ func SetDefaults_ObjectFieldSelector(obj *v1.ObjectFieldSelector) {
 }
 func SetDefaults_LimitRangeItem(obj *v1.LimitRangeItem) {
 	// for container limits, we apply default values
-	if obj.Type == v1.LimitTypeContainer {
-
+	// for pod limits, we apply default values if PodLevelResources feature is enabled
+	if obj.Type == v1.LimitTypeContainer || (obj.Type == v1.LimitTypePod && utilfeature.DefaultFeatureGate.Enabled(features.PodLevelResources)) {
```

Comment:
I don't think these defaulting rules should apply to the pod. Pod-level limit range Max & Min have meaning today, without `PodLevelResources`. Setting `Default` values here would then apply Defaulting behavior, which might not be desired. IMO we shouldn't default-the-defaults for pod-level LimitRangeItems, only allow  them to be manually set.

Reply:
Done, removed the defaulting logic for the defaults.